example/src/main/scala/com/avast/sst/example/service/RandomService.scala

```diff
@@ -0,0 +1,25 @@
+package com.avast.sst.example.service
+
+import doobie.implicits._
+import doobie.util.transactor.Transactor
+import zio.Task
+import zio.interop.catz._
+
+trait RandomService {
+
+  def randomNumber: Task[Double]
+
+}
+
+object RandomService {
+
+  def apply(transactor: Transactor.Aux[Task, _]): RandomService = new RandomService {
```

Comment:
why not just `HikariTransactor[Task]`?

Reply:
I don't think the service needs to know about the implementation of `Transactor`.

Reply:
Oh that's right! good point.
in that case would `Transactor[Task]` also work? (`Transactor.Aux[Task, _]` is a bit ugly...)